src/main/java/seedu/address/model/Model.java

```diff
@@ -91,4 +91,21 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /** Returns an unmodifiable view of the pinned person list */
+    ObservableList<Person> getPinnedPersonList();
+
+    /**
+     * Adds the {@code person} in the pinned contact list.
+     *
+     * @param person The {@code person} in the contact list to be pinned.
+     */
+    void addPinnedPersonList(Person person);
```

Comment:
Do you think it would be better to use `pinPerson(Person)` instead? Though I also know that the `add` prefix makes it clear.

Reply:
I think it is ok to leave it as `addPinnedPersonList` as `pinPerson` sounds more of an action done by logic then by model.

Reply:
That's true, makese sense :+1: